api/client/samples/crop_w_series/soybean_w_drought_index.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>Here you might want to replace <code>drought indicator</code>  with <code>Gro Drought Index</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>